web/main/templates/export/tbd.html

```diff
@@ -2,7 +2,7 @@
   <div class="SectionNumber" data-custom-style="Section Number" data-toc-idx="{{ index }}">{{ node.ordinal_string }}</div>
   <div class="SectionTitle" data-custom-style="Section Title">{{ node.title }}</div>
```

Comment:
I'm guessing title, subtitle, headnote, etc. all need a `|cut:"&nbsp;"` too? Bummer we have that stuff around in our fields..... Maybe unavoidable with the fancy GUI text editor......

Reply:
Yeah— kind of a bummer. Makes a night-and-day difference with some of the exported text, though.

Reply:
FYI— I changed this significantly. I now filter out &nbsp; entities with a single .replace operation as it's headed towards the export. The elements I had &nbsp;s in to keep them from getting stripped— section headers and stuff— I put a text placeholder in those and swapped them in a subsequent .replace().